WebRoot/js/zimbraMail/share/view/dialog/ZmDumpsterDialog.js

```diff
@@ -173,13 +171,15 @@ ZmDumpsterListView = function(parent, controller) {
 		view: this._getViewId(),
 		headerList: this._getHeaderList(),
 		type: this._getType(),
-		parentElement: (parent._htmlElId + "_listview")
+		parentElement: (parent._htmlElId + "_listview"),
+		className:"ZOptionsItemsListView DwtListView"
```

Comment:
nit: for consistency, space after colon (:)

Reply:
@plobbes , fixed here: https://github.com/Zimbra/zm-web-client/pull/207/commits/79380b765629b47505727835cd5b581fdf662dab

Thanks